VogtFactory2/Pet.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory2
+{
+    abstract class Pet
+    {
+        string Name;
+        string Sound;
```

Comment:
You're not using these variables at all. Get rid of them if you don't plan to use. 

Also an important concept that my managers/teammates have really drilled in over the past year or two..

Don't include code unless its absolutely required. Additional code can create confusion when you come back.. (why was this hear?) and just adds more of a burden to maintain that code.